src/NuGet.Clients/PackageManagement.VisualStudio/Utility/VSVersionHelper.cs

```diff
@@ -7,8 +7,19 @@
 
 namespace NuGet.PackageManagement.VisualStudio
 {
-    internal static class VSVersionHelper
+    public static class VSVersionHelper
     {
+        /// <summary>
+        /// Used to check if the code is being executed in dev14/VS 2015
+        /// </summary>
+        /// <param name="dte">DTE object used to get version</param>
+        /// <returns><code>bool</code> indicating is the code is being executed from within dev14/VS 2015</returns>
+        public static bool IsDev14(DTE dte)
```

Comment:
Even more because this is now public, I'd expect an assertion that this is running on the UI thread.  Did you not do this because NuGetPackage may be instantiated off the main thread?